lib/gui/models/store.js

```diff
@@ -287,10 +287,24 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         throw errors.createError('Missing image size');
       }
 
-      if (!_.isNumber(action.data.size)) {
+      if (!_.isPlainObject(action.data.size)) {
         throw errors.createError(`Invalid image size: ${action.data.size}`);
       }
 
+      const MINIMUM_DRIVE_SIZE = 0;
```

Comment:
This should probably be `MINIMUM_IMAGE_SIZE` because it's actually the sizes of the images that we're checking here.